src/main/java/com/epam/test_generator/entities/api/JwtTokenBuilder.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.test_generator.entities.api;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTCreator;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.epam.test_generator.entities.User;
+import org.springframework.core.env.Environment;
+
+import java.io.UnsupportedEncodingException;
+
+public class JwtTokenBuilder {
```

Comment:
And since this class does not have state it might be useful to convert it to spring bean, by annotating it as `@Service`, renaming to JwtTokenService and injecting environment.